website/docs/r/search_index.html.markdown

```diff
@@ -197,6 +197,7 @@ EOF
 * `synonyms` - Synonyms mapping definition to use in this index.
 
 * `fields` - Array of [Fields](https://www.mongodb.com/docs/atlas/atlas-search/field-types/knn-vector/#std-label-fts-data-types-knn-vector) to configure this `vectorSearch` index. It is mandatory for vector searches and it must contain at least one `vector` type field. This field needs to be a JSON string in order to be decoded correctly.
+* `status` - (Computed) Current status of the index.
```

Comment:
for consistency with our docs, should we define an Arguments section and place the computed attribute there? example: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project#attributes-reference

Reply:
thx, done